tests/integration/runners/test_fileserver.py

```diff
@@ -161,6 +163,10 @@ def test_file_list(self):
         self.assertIsInstance(ret['return'], list)
         self.assertTrue('grail/scene33' in ret['return'])
 
+    # Git doesn't handle symlinks in Windows. See the thread below:
+    # http://stackoverflow.com/questions/5917249/git-symlinks-in-windows
+    @skipIf(salt.utils.is_windows(),
+            'Git doesn\'t handle symlinks properly on Windows')
```

Comment:
Hmm, my fix won't work. In that case, @twangboy can you update the skip message to say something like "Git for Windows does not preserve symbolic links when cloning"?